src/regex.js

```diff
@@ -5,12 +5,17 @@ export const testRegex = _.curry((regex, str) => (new RegExp(regex)).test(str))
 export const makeRegex = options => text => RegExp(text, options)
 export const makeAndTest = options => _.flow(makeRegex(options), testRegex)
 
-export const matchAnyWord = _.flow(
+const anyWordToRegexp = _.flow(
   _.words,
-  _.map(makeAndTest('gi')),
-  _.overSome
+  _.map(x => `(?=.*${x})`),
```

Comment:
I'm pretty sure we don't need lookahead for this - just joining words with `|` should be fine. Getting rid of this line should do it.

Reply:
I was a bit skeptical about the need of `lookahead` here ... 
Anyways, corrected.